Source/System.Management/Pash/Implementation/ExecutionVisitor.cs

```diff
@@ -212,6 +213,17 @@ object EvaluateBinaryExpression(BinaryExpressionAst binaryExpressionAst)
             }
         }
 
+        private static bool IsInstanceOfType(object leftOperand, object rightOperand)
+        {
+            var type = rightOperand as Type;
+            if (type == null)
+            {
+                throw new InvalidOperationException(@"The right operand of '-is' must be a type.");
```

Comment:
@ygra, I think that some tools such as Resharper will throw a warning here mentioning "possibly localizable string". Making a literal verbatim is really a way to *suppress* this warning. So:

a) I don't think we generally use this practice in Pash code
b) *that's really a localizable string*, so it would be improper to just suppress the warning. If we one day will ultimately decide to localize Pash (what's unlikely but possible), we'll need to localize it.

So I don't think that you should use verbatim string literal here. Better get rid of that.